field_parser.go

```diff
@@ -21,6 +21,7 @@ const (
 	optionalLabel    = "optional"
 	swaggerTypeTag   = "swaggertype"
 	swaggerIgnoreTag = "swaggerignore"
+	uriTag           = "uri"
```

Comment:
I suggest the parameter name should be `ignoreTags` and should accept a comma-separated list of tags.